src/main/scala/io.scala

```diff
@@ -0,0 +1,47 @@
+package ohnosequences.db.rna16s
+
+import ohnosequences.db.rnacentral.Entry
+import ohnosequences.fastarious._, fasta._
+import ohnosequences.files.Lines
+
+case object io {
+
+  // Functions for generating a FASTA sequence out of an RNACentral entry.
+
+  /** join the set of original RNACentral FASTA headers */
+  private val joinDescriptions: Entry => String =
+    _.sequenceAnnotations.map(_.description).mkString(" | ")
+
+  /** FASTA sequence with all headers joined. */
+  val entryToFASTA: Entry => FASTA =
+    entry =>
+      FASTA(
+        Header(s"${entry.rnaSequence.rnaID} ${joinDescriptions(entry)}"),
+        // All upper case, U -> T
+        Sequence(entry.rnaSequence.sequence.toUpperCase.replace('U', 'T'))
+    )
+
+  /** Recieves an Iterator over the lines of a file, and reads them into
+    * a [[Mappings]] structure RNAID -> [TaxID]
+    */
+  def deserializeMappings(lines: Lines): Mappings =
+    lines.map { line =>
+      val values = line.split("†")
```

Comment:
It's ok for now, I would not change it, but maybe we could add a `serialOptions` case class that contain the separators we expect.

Reply:
I agree